intake/source/base.py

```diff
@@ -248,19 +250,22 @@ def configure_new(self, **kwargs):
         the original entry definition in a catalog **if** this source was originally
         created from a catalog.
         """
-        try:
-            obj = self.entry(**kwargs)
-            obj._entry = self.entry
+        if self._entry is not None:
+            obj = self._entry(**kwargs)
+            obj._entry = self._entry
             return obj
-        except NoEntry:  # no entry
+        else:
             kw = self._captured_init_kwargs.copy()
             kw.update(kwargs)
             return type(self)(*self._captured_init_args, **kw)
 
     __call__ = get = configure_new  # compatibility aliases
 
     def describe(self):
-        return self.entry.describe()
+        if self._entry is None:
+            raise NoEntry("Source was not made from a catalog entry")
```

Comment:
So is this really the *only* user method of the entry that would get used, aside from making more sources?